src/sgp/SGP.cc

```diff
@@ -566,7 +566,7 @@ void TerminationHandler()
 		{
 		}
 	}
-	SLOGE(errorMessage.c_str());
```

Comment:
I think you can skip the `{}` formatter, i.e.:

```c
STLOGE(errorMessage);`
```

Reply:
True. Fixed.